PSICover/PSICover.cs

```diff
@@ -183,6 +234,52 @@ class Analyzer {
          html = html.Replace ("\u00ab", "&lt;").Replace ("\u00bb", "&gt;");
          File.WriteAllText (htmlfile, html);
       }
+      var cSum = mSum.OrderBy (a => a.Percent).ToList ();
+
+      string sumcov = "";
+
+      for (int i = 0; i < cSum.Count; i++) {
+         sumcov += $"""

```

Comment:
Instead of +=, you could use string.join

Reply:
![image](https://github.com/SasiTMM/PSI/assets/125174306/8570af15-4f0f-44ca-9f72-3ea88a2c809f)

Reply:
instead of string.join, used the StringBulider

![image](https://github.com/SasiTMM/PSI/assets/125174306/f6100f8f-74c1-44c1-8444-1280c51a6300)